src/features/posts/components/user-label.tsx

```diff
@@ -0,0 +1,12 @@
+import { UserLabelSchema } from "../types/user-label"
+
+const userImg = "https://img.freepik.com/premium-photo/chita_827316-164.jpg"
```

Comment:
USER_IMG, bo to stała i możesz ją trzymać w oddzielnym pliku, chyba że tylko ten komponent to wykorzystuje